### DISCUSSION TITLE
Moving Database to Different Organization?

### DISCUSSION THREAD

**Discussion by fdarian** [planetscale/discussion#476] [FAQs] [answered]
Hi, quick question regarding Database, can it be moved to another Organization?

**Comment by orware** [comment, marked as answer]
Hello @farreldarian,

Moving a database directly from one organization to another one isn't a point-and-click operation that can be completed directly via the web UI or CLI at this time and the same tips below can also be useful for users that would like to move a database to another region too.

Currently, this process would require the use of our `pscale` CLI and the `pscale database dump` and `pscale database restore-dump` [commands](https://planetscale.com/docs/reference/database) and since the process may take a bit of time depending on the size of your database, you would likely need to schedule some related downtime for your application as well.

The good thing though is that you can run through most of the procedure below several times to practice, allowing you to get a general sense for how long the entire procedure will take.

Here are the steps you would need to complete.
 
First, create the new database in the new organization, ideally with the same name.
 
Open a terminal and dump your database locally: 

```
pscale database dump <CURRENT_DATABASE_NAME> <BRANCH_NAME> --output="<DIRECTORY_FOR_BACKUP>" --org="<ORIGINAL_ORGANIZATION>"
```

---
If your new database will have the same exact name you can skip this step, but if they will be different you currently would need to rename the files in `<DIRECTORY_FOR_BACKUP>` since they get automatically prefixed with the database name and the import will fail if you don't adjust the filenames with the new database name beforehand.

For example, the files will be named something like this:

```
<CURRENT_DATABASE_NAME>.<TABLE_NAME>-schema.sql
```

And you will want to rename all of the files in the dump folder to have the new database name if it is not the same as the existing one:

```
<NEW_DATABASE_NAME>.<TABLE_NAME>-schema.sql
```

---

To restore your database dump to the new database you would use the following command: 

```
pscale database restore-dump <NEW_DATABASE_NAME> <BRANCH_NAME> --dir="<DIRECTORY_FOR_BACKUP>" --org="<NEW_ORGANIZATION>"
 ```

You would need to repeat these steps for all branches of your database, but you can focus on your primary production branch only if that is all you need. 

 
Hopefully the method above helps to get your database over to the new organization relatively quickly!

**Reply by fdarian** [reply]
Hi Omar,

Thanks for the answer! I'll try to follow the steps.
